index.html

```diff
@@ -187,6 +187,22 @@ <h6>Skills</h6>
 
         </div>
     </section>
+    <section class="contact-section">
```

Comment:
- [x] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.